docs/general/configuration_options.md

```diff
@@ -9,6 +9,8 @@ preferably inside an initializer.
 
 - `adapter`: The [adapter](adapters.md) to use. Possible values: `:attributes, :json, :json_api`. Default: `:attributes`.
 - `serializer_lookup_enabled`: When `false`, serializers must be explicitly specified. Default: `true`
+- `default_includes`: What relationships to serialize by default.  Default: `'*'`, which includes one level of related
+  objects. See [includes](adapters.md#included) for more info.
```

Comment:
You're right. What we're actually missing here is a general documentation of the `include` option for all the adapters.